packages/dds/sequence/src/test/generateSharedStrings.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 // eslint-disable-next-line import/no-internal-modules
-import { SnapshotLegacy as Snapshot } from "@fluidframework/merge-tree/dist/test";
+import { SnapshotLegacy as Snapshot } from "@fluidframework/merge-tree/test";
```

Comment:
Better to update the test lint rules to allow test imports than to keep line disable.
x4